### DISCUSSION TITLE
PropertySet edit of IFC project

### DISCUSSION THREAD

**Discussion by Mindmelel** [IfcOpenShell/IfcOpenShell#4571] [General]
I have IFC model with the existing PropertySet and properties and I am looking for ways to add additional Properties to this specific PropertySet. 

I could not find any examples regarding this question so maybe someone could lead me to possible solutions or maybe there are non.

**Comment by aothms** [comment]
Do you want to do this programmatically or manually? 

e.g https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.7.0/src/ifcopenshell-python/ifcopenshell/api/pset/edit_pset.py#L82
